source/lex.tex

```diff
@@ -824,7 +824,7 @@
 \end{footnote}
 operators, and other separators.
 \indextext{whitespace}%
-Blanks, horizontal and vertical tabs, newlines, formfeeds, and comments
+Comments, \unicode{0020}{space}s, \unicode{0009}{character tabulation}s and \unicode{0009}{line tabulation}s, \unicode{000c}{form feed}s, and newlines
```

Comment:
I don't think plural works for the way we present Unicode chars.  Please rephrase.

Also, we seem to use "new-line" elsewhere (e.g. near the changes above). Note the hyphen.

Reply:
I specifically copied newlines from the original list, as I had seen both terms in the standard.  Now the I search again, there are only 4 remaining, and half are in code comments.  I will fix this one here, and file a separate PR to fix the rest.

Reply:
Rewording for plurals attempted.

Reply:
And it turns out the remaining three uses of "newline" are all in library clauses, which outnumber the one use of "new-line" in the library, so I will defer that question for now.  The fix for new-line above is now in this PR though.